### DISCUSSION TITLE
Connect my dabase with javascript

### DISCUSSION THREAD

**Discussion by danielalmeidabnu** [pawelsalawa/sqlitestudio#4464] [Q&A]
Hi, so i'm new to  using sqllite studio, but its a simple question how to i connect my database with javascript so i can make updates and creates.

**Comment by pawelsalawa** [comment]
You probably need to ask on Stack Overflow. I'm not expert in JavaScript.

**Comment by bobOnGitHub** [comment]
Normally: [Node](https://nodejs.org/en/), [better-sqlite3](https://www.npmjs.com/package/better-sqlite3)
Interestingly: [Sql.js](https://sql.js.org/#/)
